.github/workflows/lint.yml

```diff
@@ -0,0 +1,17 @@
+name: Lint
+
+on:
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  lint_proto_files:
+    runs-on: macos-latest
```

Comment:
Not a major - but AFAIK macos is quite expensive on github (like 10x more expensive than Linux) so would this work on Linux instead?

Reply:
Thanks for this. I'd understood that usage of github actions are [free](https://docs.github.com/en/billing/managing-billing-for-github-actions/about-billing-for-github-actions#about-billing-for-github-actions) for public repos (which this one is).

If we're talking about resources required then the macos-latest uses some [extra](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources) hardware to execute, although the execution times of the corresponding action don't appear significantly slower than others in this codebase.

`macos-latest` was chosen for the convenience of being able to `brew install...` and is used in other [projects](https://github.com/search?q=org%3Aguardian%20macos-latest&type=code) too. Perhaps if we wanted to make a change we could get the tool from source by setting up the workflow with Go.

Reply:
ah okay got you - if github actions are free for public repos then that completely negates my concern so all good :)

Reply:
Ah, no worries. I see your comment below